src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/IProjectHotReloadSessionCallback.cs

```diff
@@ -16,4 +17,15 @@ public interface IProjectHotReloadSessionCallback
 
         IDeltaApplier? GetDeltaApplier();
     }
+
+    internal interface IProjectHotReloadSessionCallback2 : IProjectHotReloadSessionCallback
+    {
+        public UnconfiguredProject? Project { get; }
+
+        public Process? Process { get; set; }
+
+        public IProjectHotReloadSession? Session { get; set; }
```

Comment:
- Accessibility on interface methods is only needed for default interface methods.
- I don't think we need setters here?

```suggestion
        UnconfiguredProject? Project { get; }

        Process? Process { get; }

        IProjectHotReloadSession? Session { get; }
```